### PR TITLE
Fix wrong documentation

### PR DESCRIPTION
**Pull request by enjoy-binbin**
This PR also change `was extreme long` to `is extremely long` to be precise.

### CODE REVIEW

**Comment by enjoy-binbin** [comment]
btw, i see this:
```
pexpire | √ | precision is in seconds
pexpireat | √ | precision is in seconds
pttl | √ | precision is in seconds
```

we says these commands precision is in seconds, is that right? i did not dive into the kvrocks source code
(redis precision is in milliseconds)

File: docs/supported-commands.md [hunk 1]
```diff
@@ -55,8 +55,8 @@
 | blpop      | √                |                                                                             |
 | brpop      | √                |                                                                             |
 | brpoplpush | X                |                                                                             |
-| lindex     | √                | Caution: linsert is O(N) operation, don't use it when list was extreme long |
-| linsert    | √                |                                                                             |
+| lindex     | √                | Caution: lindex is O(N) operation, don't use it when list was extreme long  |
+| linsert    | √                | Caution: linsert is O(N) operation, don't use it when list was extreme long |
```
Comment (torwig) [inline docs/supported-commands.md:59]: Good catch, @enjoy-binbin! Additionally, you can change `was extreme long` -> `is extremely long`  to be precise.

**Comment by git-hulk** [comment]
Hi @enjoy-binbin, thanks for your contribution first.

> we says these commands precision is in seconds, is that right? i did not dive into the kvrocks source code

Yes, that's right. Currently, Kvrocks stores the expiration time in 32-bit, so it cannot support the millisecond now. Support those commands are for compatible with Redis, and  @PragmaTwice is working on supporting 64-bit expiration time in this PR: https://github.com/apache/incubator-kvrocks/pull/1342.

File: docs/supported-commands.md (hunk 1 above)
Comment (enjoy-binbin) [reply docs/supported-commands.md]: thanks, updated it.

**Review by torwig — APPROVED** [review]
LGTM

**Review by git-hulk — APPROVED** [review]

**Pull request merged**